### ISSUE TITLE
Example for Deserializer uses `bytes_read` method which doesn't exist

### ISSUE THREAD

**Issue by joshtriplett** [bincode-org/bincode#363]
The sample code for bincode::Deserializer uses a `bytes_read`, which doesn't exist and appears to have been removed years ago.

**Comment by dtolnay** [comment]
The `Deserializer` documentation is still wrong as of current master.

**Comment by dtolnay** [comment]
The `Deserializer` documentation is still wrong as of current master.

**Comment by rillian** [comment]
The `bytes_read` reference was removed in b9e3a8001ca1c7b01a39a906d8ce566c035b5be6. I think this can be closed.

**Comment by VictorKoenders** [comment]
I'm okay with waiting on keeping this issue open until bincode 2 has been properly released. Currently we're still in 2.0-alpha and could possibly do 1.x releases

**Comment by sffc** [comment]
The `bytes_read` function would be useful when processing a slice that could have multiple serialized objects. It would be nice to have it back.

**Comment by sffc** [comment]
Prior art in `postcard`: https://docs.rs/postcard/latest/postcard/struct.Deserializer.html#method.finalize

**Comment by VictorKoenders** [comment]
In bincode 2 all the `_from_slice` functions return the amount of bytes read: https://docs.rs/bincode/2.0.0-rc.3/bincode/fn.decode_from_slice.html

**Comment by ghost** [comment]
Since 2.0 has released there will be no further doc updates to the 1.x branch. If this is still an issue on the 2.0 branch feel free to open a new issue.
